cloud-aws/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/view/AwsRdsDbParameterGroupView.java

```diff
@@ -9,6 +9,9 @@
 
 public class AwsRdsDbParameterGroupView {
 
+    @VisibleForTesting
+    static final String DEFAULT_FAMILY_VERSION = "10";
```

Comment:
Is it possible to use the parameter cb.aws.rds.version here, instead of this constant?

Reply:
this should be removed